app/build.gradle

```diff
@@ -46,22 +46,22 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:27.1.0'
-    compile 'com.android.support:design:27.1.0'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:design:27.1.1'
     implementation 'protect:android-ffmpeg:release@aar'
-    compile 'com.crystal:crystalrangeseekbar:1.1.3'
-    compile 'commons-io:commons-io:2.5'
-    compile group: 'com.google.guava', name: 'guava', version: '20.0'
-    compile 'com.nononsenseapps:filepicker:4.2.1'
-    implementation 'com.simplecityapps:recyclerview-fastscroll:1.0.9'
+    implementation 'com.crystal:crystalrangeseekbar:1.1.3'
+    implementation 'commons-io:commons-io:2.6'
+    implementation group: 'com.google.guava', name: 'guava', version: '25.1-android'
+    implementation 'com.nononsenseapps:filepicker:4.2.1'
+    implementation 'com.simplecityapps:recyclerview-fastscroll:1.0.18'
     implementation group: 'org.javatuples', name: 'javatuples', version: '1.2'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.8.11.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.6'
 
     implementation 'ch.acra:acra:4.11'
 
-    testCompile 'junit:junit:4.12'
-    testCompile "org.robolectric:robolectric:3.6.1"
+    testImplementation 'junit:junit:4.12'
+    testImplementation "org.robolectric:robolectric:4.0-alpha-2"

```

Comment:
I would like to avoid using alpha releases. Should we wait on this one?

Reply:
ill set it to 3.8, which is the latest stable